### PR TITLE
Update Cloudstream

### PR DESCRIPTION
**Pull request by DoYouNotice**
there is also this repo with the newest official update that was normal https://github.com/rereleased/release/releases/tag/v3.1.2

### CODE REVIEW

**Comment by XXDRD** [comment]
183259075-141b1939-f26a-47c1-8eca-c9f584a4b0c1
sad

File: README.md [hunk 1]
```diff
@@ -1416,12 +1416,12 @@ Currently being reworked as Neo Launcher.
 
 ## Streaming
 ### CloudStream
-<img alt="CloudStream" height="64" src="https://github.com/LagradOst/CloudStream-3/blob/master/app/src/main/ic_launcher-playstore.png?raw=true">
+<img alt="CloudStream" height="64" src="https://apt.izzysoft.de/fdroid/repo/com.lagradost.cloudstream3/en-US/icon.png">
```
Comment (XXDRD) [inline README.md:1419]: change image to https://raw.githubusercontent.com/recloudstream/cloudstream/master/app/src/main/ic_launcher-playstore.png

**Review by XXDRD — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DoYouNotice) [reply README.md]: done

**Comment by Psyhackological** [comment]
Thanks @XXDRD. Good pull requests @DoYouNotice. Squashing and merging.

**Pull request merged**